src/routes/Speakers.svelte

```diff
@@ -37,17 +37,16 @@
 <div class="bg-slate-100 w-full px-8 py-28">
 	<div id="speakers" class="container mx-auto flex flex-col items-center text-center max-w-5xl">
 		<h2 class="mb-16">{$_('speakers.title')}</h2>
-
-		<div class="grid md:grid-cols-4 gap-16">
-			{#each speakers as speaker (speaker.name)}
-				<div class="flex flex-col">
-					<a href="/speakers/{speaker.slug}">
-						<Avatar src={speaker.imgSrc} width="w-48" height="h-48" class="mb-4" />
-					</a>
-					<h4>{speaker.name}</h4>
-					<p>{speaker.company}</p>
-				</div>
-			{/each}
-		</div>
+	</div>
+	<div class="grid md:grid-cols-3 gap-16">
```

Comment:
Moving the speakers outside the `#speakers` div doesn't work, as now they spread wider than the rest of the content of the website. I think, we have to find a different solution. @mkretz, should we check together tomorrow?